### ISSUE TITLE
Exporting users warns magic.mime is missing

### ISSUE THREAD

**Issue by marcbria** [pkp/docker-ojs#49]
When exporting users, log shows the following warning:

```
PHP Warning:  finfo_open(/etc/magic.mime): failed to
 open stream: No such file or directory in /var/www/html/lib/pkp/classes/core/PKPString.inc.php on line 318, referer: http://ada.uab.cat:811
7/languesparole/management/importexport/plugin/UserImportExportPlugin
```

This warning raises when OJS calls fileinfo functions: https://www.php.net/manual/es/class.finfo.php

A preliminary search points that libmime is missing but installing it via "file" package ("apk add --no-cache file") makes no real difference.

When running "file -version" I got the path of the magic file (not magic.mime but I suspect will do the job) and it's /usr/share/misc/magic. 

A possible solution would be change the dockerfile to add "file" and link "/etc/magic.mime" to "/usr/share/misc/magic" but I'm not sure about side effects.

I'm not sure if this only happens with 3.2.1-4 over php 7.3 or is a general issue.

In short: although this is just a warning and I can see any real impact in the application functionality... more research is required.

**Comment by marcbria** [comment]
Ubuntu includes a /etc/magic.mime that is empty.
A touch on /etc/magic.mime removes the warning but would be nice to have mime support.

Installing imagemagick (apk --update add imagemagick) adds a new mime folder.
Linking /usr/share/mime/magic to /etc/magic.mime also fixes the warning.

Installing libmagic to test... and I can't see any new new magic.mime file.

Not sure about what solution would be better.

Kind of related with #33

**Comment by marcbria** [comment]
Looks like apache includes a mime file.
Linking /etc one to apache fixes the issue:

$ ln -s /etc/apache2/magic /etc/magic.mime 

Anyway, as far as we are going to move from alpine to debian-slim it won't be required anymore.
